waiter/src/waiter/scheduler/scheduler.clj

```diff
@@ -8,7 +8,7 @@
 ;;       The copyright notice above does not evidence any
 ;;       actual or intended publication of such source code.
 ;;
-(ns waiter.scheduler
+(ns waiter.scheduler.scheduler
```

Comment:
Why not leave everything you moved to `waiter.scheduler.scheduler` in `waiter.scheduler`?  Prior art:

https://github.com/clojure/clojure/tree/master/src/clj/clojure, `core.clj` and `core` folder

Reply:
I agree with @sradack

Reply:
Undid move of `scheduler.clj`